packages/taquito/src/prepare/prepare-provider.ts

```diff
@@ -219,7 +221,12 @@ export class PrepareProvider implements PreparationProvider {
             ...op,
             period: currentVotingPeriod?.voting_period.index,
           };
-
+        case OpKind.SMART_ROLLUP_ADD_MESSAGES:
```

Comment:
Seems to return the same thing as line 189-193, would be great to add a line of `case OpKind.SMART_ROLLUP_ADD_MESSAGES:` below line 188 to reduce repetition